.travis.yml

```diff
@@ -15,12 +13,22 @@ before_install:
 script:
 - docker build --target test --tag my-test-image .
 - docker run my-test-image ./todo_app/test/test_view_model.py
-- docker run my-test-image ./todo_app/test/test_app.py
+- > 
+    docker run 
+    -e user_name=$MONGODB_USER_NAME
+    -e password=$MONGODB_PASSWORD
+    -e mongo_url=$MONGO_URL
+    -e database_name=$DATABASE_NAME
+    -e collection_name=$COLLECTION_NAME
+    my-test-image ./todo_app/test/test_integration.py
```

Comment:
Your integration tests shouldn't need real credentials - you mock out Mongo. If you make the change I've suggested at the bottom of `app.py` I think you can avoid passing all these environment variables in.